app/src/db/models/conversation.py

```diff
@@ -28,3 +30,7 @@ class UserSession(Base, TimestampMixin):
     lai_thread_id: Mapped[str | None] = mapped_column(
         comment="LiteralAI's thread ID corresponding to session_id"
     )
+    chat_messages: Mapped[list[ChatMessage]] = relationship(
+        backref="ChatMessage.session_id",
```

Comment:
To avoid backref (which is deprecated) I think you can do:
```suggestion
        back_populates="ChatMessage.session_id",
```
here and then add `session: Mapped["UserSession"] = relationship("UserSession", back_populates="chat_messages")` to ChatMessage above

Reply:
updated